lib/bucky/test_equipment/pageobject/base_pageobject.rb

```diff
@@ -55,7 +55,8 @@ def find_elem(method, value)
 
           elem = @driver.find_elements(method_name, value)
           raise_if_element_empty(elem, method_name, value)
-          elem
+          Selenium::WebDriver::Element.class_eval { define_method('[]') { |num| elem[num] } }
```

Comment:
I don't think it's a good idea using define_method in every find_elem.
Maybe adding like this at the first of BasePageObject class.
```
Selenium::WebDriver::Element.class_eval { define_method('[]') { |num| self[num] } }
```

Reply:
Fixed!